doc/radosgw/frontends.rst

```diff
@@ -46,6 +46,8 @@ Options
 ``ssl_certificate``
 
 :Description: Path to the SSL certificate file used for SSL-enabled endpoints.
+              If path is prefixed with ``config://``, the certificate will be
+              pulled from the ceph monitor ``config-key`` database.
```

Comment:
can we document the security considerations here? aren't these keys being sent over the wire in cleartext unless messenger encryption is on? how can we make it impossible for users to get this wrong?

a link to some docs about setting up the keys would be helpful too

Reply:
Maybe we can require that if this is set then `client_modes` and possibly `mon_client_modes` are set to `secure` only?

Reply:
Also, we should probably enforce `secure` mode all on communication to/from mon and mgr. But that's maybe a different discussion. See https://github.com/ceph/ceph/pull/33288 also.